js/exporter/jspdf/export.js

```diff
@@ -125,24 +126,30 @@ export const Export = {
         }
     },
 
-    setColumnWidths: function(autoTableOptions, columns, keepColumnWidths) {
+    setColumnWidths: function(jsPDFDocument, autoTableOptions, columns, keepColumnWidths) {
         const columnStyles = autoTableOptions.columnStyles;
 
         for(let i = 0; i < columns.length; i++) {
             columnStyles[i] = columnStyles[i] || {};
 
-            const columnWidth = columns[i].gridColumn.width;
-            if(keepColumnWidths && (typeof columnWidth === 'number') && isFinite(columnWidth)) {
-                columnStyles[i].cellWidth = this.convertPixelsToPoints(columnWidth);
+            const isColumnWidthDefined = isDefined(columns[i].gridColumn.width);
+            if(keepColumnWidths && isColumnWidthDefined) {
```

Comment:
&& typeUtils.isNumeric ?
It will be explicit mark that we know about "A CSS-accepted column width measurement (for example, "55px", "80%" and "auto"" and will not support it for now. New item in our checklist?

Reply:
I want to clarify the specifics of the code logic

**columns[i].width** has a value that reflects the value in the browser. The value is 'Number' in "px".
**columns[i].gridColumn.width** has a value that reflects the value in the dxDataGrid config ( 50, '55px', '55pt', '50%' ... ) or 'undefined'

the logic is:
```
var isColumnWidthDefinedInOptions = isDefined(columns[i].gridColumn.width);  // Check if the width for column via the current index is defined in DataGrid options
autoTableOptions.columnStyle[i] = isColumnWidthDefinedInOptions
    ? columns[i].width // set the width for column via the current index converted to pixels
    : 'auto';
```
Thus, the size specified by the user in the required unit of measurement will be correctly obtained in pixels.